Mathlib/Algebra/Category/GroupWithZeroCat.lean

```diff
@@ -73,8 +73,9 @@ instance groupWithZeroConcreteCategory : ConcreteCategory GroupWithZeroCat where
     map := fun f => f.toFun }
   forget_faithful := ⟨fun h => FunLike.coe_injective h⟩
 
--- porting note: added
-@[simp] lemma forget_map (f : X ⟶ Y) : (forget GroupWithZeroCat).map f = f := rfl
+-- -- porting note: added
+-- @[simp] lemma forget_map (f : X ⟶ Y) : (forget GroupWithZeroCat).map f = f := rfl
```

Comment:
What happened here?

Reply:
I optimistically decided the lemma wasn't needed, and was right. If we proceed we should restore it.